### DISCUSSION TITLE
manual installation directions question

### DISCUSSION THREAD

**Discussion by ObviousInRetrospect** [SpenceKonde/megaTinyCore#766] [Q&A]
The manual installation directions say:

"You must be using a copy of the Arduino IDE that has never had an AVR board definition package installed on it (typically this means the .zip archive, extract, and create a portable folder inside before first run)"

Why is this the case? What goes wrong when it already exists?

I know I have done manual installation of esp32 without that being the case.

**Comment by SpenceKonde** [comment]
That bug may or may not have been fixed. At one point, it would overwrite
the version of avr-gcc used by other cores potentially breaking other
cores.

Only AVR cores had that effect. 

Personally, I have one version of the IDE installed that has never used board manager which I directly update the builtin compiler on, and portable installations for when I am forced to use board manager for testing. Three of them are 1.8.13

**Reply by ObviousInRetrospect** [reply]
oh so if the only cores I use these days are megaTinyCore, DxCore, ESP32, and STM32 do I not care? (mega328s suck)

**Comment by SpenceKonde** [comment]
Correct. And I thi k the bug was fixed by 1 8.13 but I have no time to
track it down

____________
Spence Konde
Azzy’S Electronics

New products! Check them out at tindie.com/stores/DrAzzy
GitHub: github.com/SpenceKonde
ATTinyCore: Arduino support for almost every ATTiny microcontroller
Contact: ***@***.***
